src/Registry.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "openzeppelin-contracts/contracts/access/Ownable.sol";
+
+/// @title  Registry contract :: Checking if a token's legit
+/// @notice This contract can be called by our main contract for checking if the ERC20 token address is legit
+
+contract Registry is Ownable {
+    mapping(address => bool) isLegit;
+
+    event TokenSetTo(address indexed, bool value);
```

Comment:
Maybe `ChangedVerificationStatus`?

also, give it names so the codegen can be smarter: `address indexed erc20, bool value`